### DISCUSSION TITLE
ABP Commercial vs ASP.NET Zero

### DISCUSSION THREAD

**Discussion by TonyValenti** [abpframework/abp#6260] [Q&A]
They both look great and they're both made by Volosoft.  Is there a product comparison matrix I can look at?  which should be used for new development?

**Comment by hikalkan** [comment]
Hi @TonyValenti, 

Actually there is a comparison chart: https://docs.abp.io/en/commercial/latest/aspnet-zero-vs-abp-commercial
For new & long-term development, we can suggest the ABP Commercial. Most of the Volosoft developers are working on it and it is more flexible, modular, modern and microservice compatible.
However, AspNet Zero has still some advantages. You compare and decide :)

**Reply by MuhammadRashed** [reply]
Yes, but the Aspnetzero is much mature, it contains all commericial module source code, and you have the framework source code, the tool (generating source code tool) is way advanced in aspnetZero than abp commercial, I advice people to use aspnetZero for the next year until the abp get the same functionallity and become much stable,
Even regarding the microservice architecture is not that easy in abp, it requires a lot of work it to make a simple poc event if you use the suite tool,
aspnetzero has a full tools for ui unlike abp commercial

**Comment by douglasbustos** [comment]
Please update this information, we need to decide buy a version.

**Reply by ismcagdas** [reply]
@douglasbustos there is only 1 item "Tenant subscription, payment & billing system" which is released with ABP Commercial 4.4. The payment module was there before but subscription functionality is added with 4.4 version. However, the subscription feature in ABP Commercial is a bit low level but provides more functionality. If you have specific questions, please send your questions to info@abp.io or write here and we will answer them.
